test/modules/TestResultCoordinator/ITestResultReport.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TestResultCoordinator
+{
+    /// <summary>
+    /// This defines the basic properties of a test result report.
+    /// </summary>
+    interface ITestResultReport
```

Comment:
>ITestResultReport [](start = 14, length = 17)

Can this interface be generic instead of having ResultType? Then ExpectedSource and ActualSource have T as type

Reply:
Every report should only support 1 result type; which is used to derive what result comparer to use.  ExpectedSource and ActualSource is just source name, e.g. "loadGen1.send" or "Relyer1.receive".